### ISSUE TITLE
Meteor integration

### ISSUE THREAD

**Issue by gbisheimer** [rsamec/react-designer-core#1]
I'm following this project progress for a while now and its excellent!!. We are developing applications using Meteor framework and we want to integrate this great tool in it for the admin interface.

Our goal is to be able to edit the application layout just by drag & drop, collaboratively in real time.

Have you integrated this utility with meteor already?

**Comment by rsamec** [comment]
No, i have not made any example with meteor yet. But it sounds to me as a great idea to have an editor with support for collaborative work in real time.

I think it is feasible - > your clients subscribes and constantly keeps it up to date with the latest information on the server (the document definition - simple json).

It would be nice, if you could add an simple example of the meteor integration to the example folder.
